test/studies/lulesh/intelVectBug/lulesh.compopts

```diff
@@ -0,0 +1,6 @@
+-M../../../release/examples/benchmarks/lulesh --fast
```

Comment:
Maybe throw `--vectorize` as well, just in case we run into more issues and stop tying it to `--fast`?